include/pyre3.14t/modsupport.h

```
@@ -118,7 +122,10 @@ static inline int _PyPyre_ArgConvert(PyObject *arg, const char **format,
         Py_ssize_t length = 0;
         if (code == 'z' && Py_IsNone(arg)) {
             text = NULL;
-        } else if (code == 'y') {
+        } else if (code == 'y' || (with_size && PyBytes_Check(arg))) {
```

Clang (14.0.6)

[note] +1, nesting level increased to 2

(clang)